### ISSUE TITLE
[Art competition] Undecidability of Mathematics

### ISSUE THREAD

**Issue by cwlo2F** [prosyslab-classroom/cs492-program-reasoning#164]
![DALL·E 2023-10-07 20 30 43](https://github.com/prosyslab-classroom/cs492-program-reasoning/assets/48020071/71c88ad1-dc82-45b5-abac-29e8b93a85af)

## Short Description
For me, [*Undecidability of Mathematics*] means "[*various trials from a finite earth to reach the infinitely far stars in the sky, including building skyscraper or launching rockets*]".

## Long Description (up to 3 sentences)
Machines, from the earth, dream of traveling mysterious lands.
Undecidability, above the sky, foretells the fate of finite mathematicians.
I, as a human, hope to keep crafting art as long as possible.

**Comment by KihongHeo** [comment]
Interesting. Does the AI generate the drawing from your prompt? "various trials from a finite earth to reach the infinitely far stars in the sky, including building skyscraper or launching rockets"

**Comment by cwlo2F** [comment]
Indeed, it is not a direct result. Using the exact prompt, I was able to get a drawing describing 'various trials' in checkboard pattern as the above drawing. With that drawing, I made variations and chose the above one, because it is more colorful and it accidentally contains a 'jumping person'. While it is from a variation, the display of the above drawing is quite similar to the first shot.
I believe that I was lucky for getting those drawings.
